src/main/java/edu/kis/powp/jobs2d/events/SelectClearPanelOptionListener.java

```diff
@@ -6,8 +6,8 @@
 import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 public class SelectClearPanelOptionListener implements ActionListener {
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		DrawerFeature.getDrawerController().clearPanel();
-	}
+    @Override
+    public void actionPerformed(ActionEvent e) {
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.